source/instance.txt

```diff
@@ -11,7 +11,16 @@ Compass Home
    :class: singlecol
 
 The Compass Home screen provides details regarding the
-MongoDB instance to which Compass is connected, including:
+MongoDB instances to which Compass is connected. 
+
+.. note::
+
+   Starting in version 1.44, you can connect to multiple MongoDB
+   instances at once through Compass. When you active multiple
```

Comment:
```suggestion
   instances at once through Compass. When you activate multiple
```
or
```suggestion
   instances at once through Compass. When you have active multiple
```

I don't know which one it is though :D

Reply:
fixed!